### PR TITLE
enh: centreon dark mode for resources status-listingPage component-

### PR DESCRIPTION
**Pull request by Noha-ElAbrouki**
centreon dark mode for resources status for listing page

### CODE REVIEW

**Comment by Noha-ElAbrouki** [comment]
here are some few screens : 
listingPageAvant
listingPageApres

**Review by bdauria — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Comment by Noha-ElAbrouki** [comment]
here are some screens for listingPage for actions downTime and acknowledge : 
 before
ListingPage

after
listingActionApres1
ListingAknowledgeAndDownTime

**Comment by Noha-ElAbrouki** [comment]
here are some screens: 
before
ListingPageActionsAvant

after 
ListingPageActions
ListingPageDownTime
ListingPageAcn

**Comment by Noha-ElAbrouki** [comment]
here are some screens:
before 
ListingPage

after
downTime and acknowledg
downTimeWithDetail
acknowledgeWithDetail

File: packages/centreon-ui/src/ThemeProvider/index.tsx [hunk 1]
```diff
@@ -49,10 +49,10 @@ export const getTheme = (mode: ThemeMode): ThemeOptions => ({
...
         }),
     action: {
-      acknowledged: '#AA9C24',
-      acknowledgedBackground: '#F7F4E5',
-      inDowntime: '#9C27B0',
-      inDowntimeBackground: '#F9E7FF',
+      acknowledged: mode === ThemeMode.light ? '#635a15' : '#F7F4E5',
+      acknowledgedBackground: mode === ThemeMode.light ? '#F7F4E5' : '#635a15',
+      inDowntime: mode === ThemeMode.light ? '#4e1358' : '#F9E7FF',
+      inDowntimeBackground: mode === ThemeMode.light ? '#F9E7FF' : '#4e1358',
```
Comment (Thebarda) [inline packages/centreon-ui/src/ThemeProvider/index.tsx:55]: You can extract all the `mode === ThemeMode.light` in a specific function like `isLightTheme`.
BTW, don't hesitate to use the `equals` function from Ramda https://ramdajs.com/docs/#equals ;)

**Review by bdauria — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**